osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -355,6 +355,14 @@ private void handleEvent(string line)
 
             switch (type)
             {
+                case LegacyEventType.Sprite:
```

Comment:
Hmm.... The above code seems like this will also match `Sample`, which is the case osu-stable was avoiding. best to also do something similar here?

Reply:
I *think* you might have misread the code. It's matching against the full word here.

Reply:
I did indeed.